Egregious-Eggnog/reactive-nutshell
# Lr styling

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

1. `git fetch --all`
2. `git checkout lr-styling`
3. look at styling and double check login and register and CRUd for ech page works


# Checklist:
